index.d.ts

```diff
@@ -35,6 +35,8 @@ type CAPIDesign = DesignType | "Immersive" | "SpecialReport" | "GuardianLabs";
 // eslint-disable-next-line @typescript-eslint/no-explicit-any
 type DesignTypesObj = { [key in DesignType]: any };
 
+type Palette = {[key: string]: any};
```

Comment:
prettier issues? the spacing isnt quite right.

Also maybe better to call this `PaletteType` to make it more obvious that it is a type?

Reply:
I went with `Pallete` to match the naming convention from `@guardian/types` where you have `Design`, `Theme`, etc.

Reply:
I _do_ think a Type suffix is useful though, especially when  dealing with prop data being used for a component where the namespace can be shared a lot